lib/ndr_import/non_tabular/table.rb

```diff
@@ -117,6 +117,8 @@ def partition_and_process_non_tabular_lines
             # This is a start line
             start_record(line)
           elsif line =~ @end_line_pattern
+            # Add the end line to the @non_tabular_record (if required) before ending the record
+            @non_tabular_record << line if @capture_end_line
```

Comment:
For symmetry with start_record / @capture_start_line, should this be handled within end_record?

Reply:
Thanks @joshpencheon, this has been done in dc7bf292af3abcff88e0b42da7b703fdd2665b91